tests/unit/dag/test_column_selector.py

```diff
@@ -270,6 +270,12 @@ def test_applying_inverse_selector_to_schema_selects_relevant_columns():
     assert result == schema
 
 
+def test_wildcard_selector_bool():
+    selector = ColumnSelector("*")
+
+    assert selector.all is True
```

Comment:
It looks like this test would have passed without this change. I'm not sure if this PR has a functional change, though the property method is clearer in terms of documenting the public interface.

Reply:
I'm right there with you on thinking this shouldn't be a functional change, but if you see the linked bug ticket, somehow `self.all` is not getting set/created on selector objects. I think this should address that issue, although I'm not entirely sure why the code wasn't working as written.